CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [unreleased]
 
+- Fix: Remove `button-heart-icon-words` to `button-with-icon-words`.
```

Comment:
Du coup, j'écrirais plutôt : 
"Replace `KarlButtonIconHeartRight` and `KarlButtonIconHeartLeft` examples with `KarlButtonIconWordsRight` and `KarlButtonIconWordsRight`".